config/i18n-tasks.yml

```diff
@@ -1,7 +1,7 @@
 # i18n-tasks finds and manages missing and unused translations: https://github.com/glebm/i18n-tasks
 
 # The "main" locale.
-base_locale: en
+base_locale: es
```

Comment:
Yeah, just a work in progress. Jessie suggested that I not wait too long to submit a pull request otherwise it will get too big. :) And yeah, you're right about the en part. I had changed it to view it in spanish locally.